assets/js/admin/orders/order-overview/_refund.js

```diff
@@ -221,7 +221,8 @@ function recalculateRefundTotal() {
 $(document.body).on( 'click', '#edd-submit-refund-submit', function(e) {
 	e.preventDefault();
 	$('.edd-submit-refund-message').removeClass('success').removeClass('fail');
-	$( this ).attr( 'disabled', true );
+	$( this ).removeClass( 'button-primary' );
+	$( this ).attr( 'disabled', true ).addClass( 'updating-message' );
```

Comment:
YES thank you I like this so much!

I _think_ the two lines could be combined if we want to be super finicky (not tested):

```js
$( this ).removeClass( 'button-primary' ).attr( 'disabled', true ).addClass( 'updating-message' );
```
but I love this.

Reply:
I had meant to test chaining those statements, but then I got too excited by the new button behavior. 🤷‍♀️